Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -1606,8 +1606,23 @@ private bool PrepareFileAccessMonitoringCommon(LoggingContext loggingContext)
             }
 
             // Untrack the globally untracked paths specified in the configuration 
+            var reverseTranslator = m_fileAccessManifest.DirectoryTranslator?.GetReverseTranslator();
             foreach (var path in m_sandboxConfig.GlobalUnsafeUntrackedScopes)
             {
+                // translate the path and untrack the translated one
+                if (reverseTranslator != null)
+                {
+                    var pathString = path.ToString(m_pathTable);
+                    var translatedPathString = reverseTranslator?.Translate(pathString);
+                    var translatedPath = AbsolutePath.Create(m_pathTable, translatedPathString);
+                  
+                    if (pathString != translatedPathString)

```

Comment:
You might have to do case insenstive string comparisson here..
At least 'ordinal' since now it is culture specific.

Reply:
Even better: compare `AbsolutePaths` (which should take care of any platform-specific requirements for whether to use case sensitive or case insensitive)